dumpling-overview.md

```diff
@@ -60,10 +60,18 @@ dumpling \
   --filetype sql \
   --threads 32 \
   -o /tmp/test \
+  -r 200000 \
```

Comment:
I think it's too large and may cause TiDB OOM.

Reply:
If we suggest TiDB uses 1GB memory, and the default threads is 4, then avg_row_length is 
1GB / 4 / 200000 = 1342. It's more than 1KB. In most conditions `avg_row_length` is less than this.